datastore/kafka/kafka.go

```diff
@@ -58,9 +60,11 @@ func NewProducer(config *kafka.ConfigMap, namespace string, reliableAckWorkers i
 		logger:            logger,
 		airbrakeHandler:   airbrakeHandler,
 		reliableAck:       reliableAckWorkers > 0,
+		deliveryChan:      make(chan kafka.Event),
+		ackChan:           ackChan,
```

Comment:
I don't understand this design for ackChan.  Wouldn't we need an ackChan per connection?

Reply:
you are right. I have disabled setting reliable ack config for now. Will remove the restriction when its supported properly

Reply:
ok, well we should complete the implementation or remove ackChan. Let's not leave code/variables around that aren't being used